opensuse-tomcat-image/src/main/docker/logback.xml

```diff
@@ -33,6 +33,9 @@
         </encoder>
     </appender>
 
+    <logger name="log4j.logger.org.apache.commons.digester.Digester" level="${CAF_LOG_LEVEL:-INFO}" additivity="false">
```

Comment:
Where did you get the logger name?  `log4j.logger.` seems a little strange since I thought we were using Logback rather than Log4J.

Reply:
I notice you've removed the `log4j.logger.` prefix now.  Did you verify that was incorrect?